src/components/views/Article/Article.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import Tags from 'views/Tags';
+import './Article.scss';
+
+class Article extends Component {
+    render() {
+        const { id, date, title, author, tags, content } = this.props.post,
+            pub_date = new Date(date * 1000);
```

Comment:
As I know defining variables with commas is bad practice. I suggest not to do it